azurecontainerapps/src/ContainerAppHelper.ts

```diff
@@ -266,7 +300,7 @@ export class ContainerAppHelper {
         try {
             const command = `containerapp env list -g ${resourceGroup} --query [0].name"`;
             const result = tl.execSync('az', command);
-            return result.code == 0 ? result.stdout : null;
+            return result.code == 0 ? result.stdout.replace(/(\r\n|\n|\r)/gm, "") : null;
```

Comment:
Nit: We should probably remove `\r` `\n` from stdout in all the methods in this file to be consistent with what we are doing in the GitHub action? Can we extract `tl.execSync` to another method and call that method everywhere else to execute commands?

Reply:
Good point. But for now, I would prefer to add this fix in the ADO task main repo later on and unblock the customer quickly.